### ISSUE TITLE
Quotes with hard newlines

### ISSUE THREAD

**Issue by snan** [makeworld-the-better-one/md2gemini#32]
Hope you are doing well makeworld♥

Here's the issue:
It's turning (note double space at end of lines)

```
> What is a good man?  
> A teacher of a bad man.  
> What is a bad man?  
> A good man's charge.
```

into

```
> What is a good man?
A teacher of a bad man.
What is a bad man?
A good man's charge.
```

**Comment by makeworld-the-better-one** [comment]
Thanks for reporting, looks like the double space is being treated as a hard line break that breaks the quote. Seems like md2gemini has quite a few issues now... I'd appreciate a PR, but otherwise I'll work on this when I can. Not my top priority unfortunately.

**Comment by makeworld-the-better-one** [comment]
Fixed in 79bc115

**Comment by snan** [comment]
Great ♥
